utils/frame/benchmarking-cli/src/lib.rs

```diff
@@ -87,6 +88,17 @@ impl CliConfiguration for BenchmarkCmd {
 		}
 	}
 
+	fn base_path(&self) -> Result<Option<BasePath>> {
+		let inner = unwrap_cmd! {
+			self, cmd, cmd.base_path()
+		};
+
+		match inner {
```

Comment:
Some comment on why we are doing this would be nice.

Reply:
done